tests/DocumentationExamplesTest.php

```diff
@@ -1865,6 +1865,8 @@ public function testQueryableEncryption(): void
             $this->markTestSkipped('Automatic encryption requires MongoDB Enterprise');
         }
 
+        $this->dropCollection();
```

Comment:
I just remembered that the entire database is dropped below:

```php
$client->selectDatabase($databaseName)->drop(['writeConcern' => new WriteConcern(WriteConcern::MAJORITY)]);
```

I think we should probably remove `$this->dropCollection();` here.

The multi-line comment below explains that the database is dropped because "setUp() only drops the collection under test", which is no longer true. I'd suggest revising that comment to say:

> Create a client without auto encryption. Drop existing data in both the keyvault and database under test. The latter is necessary to clean up any internal collections for queryable encryption.

Note: this highlights an issue I may have mentioned in a previous comment about your create and drop helpers not handling encrypted collections. We don't need to deal with this now. You can consider making a new PHPLIB ticket to revisit that in the future (and just mark it as related to PHPLIB-816). The solution for that will probably entail creating a `MongoDB\Collection` object to use _its_ create/drop helpers instead using the specific operation classes as you do now.

----

For additional context: this CSFLE test is unique among other tests in the file in that it is _not_ parsed into the server manual. It was only added as a reference for the docs team in [PHPLIB-863](https://jira.mongodb.org/browse/PHPLIB-863).

Reply:
The `dropCollection` is here to ensure the collection is dropper on teadDown. I'll add a comment to explain the duplicate with the database drop.

This test cannot run until https://github.com/mongodb/mongo-php-library/pull/1064

Reply:
> The dropCollection is here to ensure the collection is dropper on teadDown. I'll add a comment to explain the duplicate with the database drop.

Ah, that makes sense. Since the metadata collections are going to be left behind anyway (per the second-to-last paragraph in my last response), I don't think it's worth addressing this now. You should make a note of it in the follow-up issue, though (and cross-reference this comment thread for context).

Reply:
I updated `dropCollection` and `cleanupCollections` so they pass the options to `Collection::drop($options)`. This effectively drops the metadata collections.